test/base.test.js

```diff
@@ -1,15 +1,33 @@
 'use strict'
 
-const { test, mock } = require('tap')
+const { test, mock, before, teardown, afterEach } = require('tap')
 const fastify = require('fastify')
 const mercurius = require('mercurius')
+const FakeTimers = require('@sinonjs/fake-timers')
 const WebSocket = require('ws')
 const cache = require('..')
 
 const { promisify } = require('util')
 const immediate = promisify(setImmediate)
 const { request } = require('./helper')
 
+let clock
+
+before(() => {
+  clock = FakeTimers.install({
+    shouldAdvanceTime: true,
+    advanceTimeDelta: 0
+  })
+})
+
+afterEach(() => {
+  clock.runAll()
+})
```

Comment:
is this needed?

Reply:
Strictly this is not required for the current tests, however if someone adds another test that leverages the fake timers they will run into issues. I thought it was best to be consistent with the clean up and reduce potential future issues.